spec/fontcustom/cli_spec.rb

```diff
@@ -26,5 +26,23 @@
         expect(File.read(css)).to match("../foo/bar/")
       end
     end
+
+    context 'single quotes' do
+      it "should generate fonts and templates with single quotes" do
+        live_test do |testdir|
+          Fontcustom::CLI.start ["compile", "vectors", "--templates", "preview", "css", "scss-rails", "--single-quotes"]
+          preview = File.join testdir, "fontcustom", "fontcustom-preview.html"
+
+          expect(Dir.glob(File.join(testdir, "fontcustom", "fontcustom_*\.{ttf,svg,woff,eot}")).length).to eq(4)
+          expect(File.exists?(preview)).to be_truthy
+
+          generated_css = File.read Dir.glob(File.join(testdir, "fontcustom", "*.css")).first
+          expect(generated_css.scan('"').count).to be 0
```

Comment:
The easiest way to assert that all " were replaced with ' is to assert there are no ". Of course that might also mean that there are no ' either, or even worse to assert, there's invalid balance of ' to enclose values. I don't know how to assert that the right amount of ' were replaced, and whether they were put in the right places.

I don't know how to do that, except by parsing the generated css for a valid CSS along with the current assertion of no " present. Any other idea?